compiler/AST/LoopExpr.cpp

```diff
@@ -214,6 +227,33 @@ void lowerLoopExprs(BaseAST* ast) {
   ast->accept(&vis);
 }
 
+
+static Expr* getShapeForZippered(Expr* tupleRef) {
+  Symbol* tupleSym = toSymExpr(tupleRef)->symbol();
+  SymExpr* tupleDef = tupleSym->getSingleDef();
+  CallExpr* move = toCallExpr(tupleDef->parentExpr);
+  INT_ASSERT(move->isPrimitive(PRIM_MOVE));
+  CallExpr* buildTup = toCallExpr(move->get(2));
+  INT_ASSERT(buildTup->isNamed("_build_tuple"));
+  // The shape comes from the first tuple component.
+  return buildTup->get(1);
+}
+
+// 'feRep', during resolution, is an iterator record for the
+// forall expression. Ensure it will get a shape.
```

Comment:
feRep is a call so it can't possibly be an iterator record, which is a type.
In any case I think you could come up with a better variable name. I don't think the abbreviation here is helping.